group_vars/all.yml

```diff
@@ -368,6 +369,17 @@ openvpn_password: secretpassword
 openvpn_provider: AWESOMEVPNPROVIDER
 openvpn_config: United-Kingdom
 
+###
+### uTorrent
+###
+utorrent_available_externally: "false"
+utorrent_config_directory: "{{ docker_home }}/utorrent/config"
+utorrent_download_directory: "{{ downloads_root }}"
+utorrent_port_http: "8111"
+utorrent_port_bt: "6881"
+utorrent_user_id: "999"
```

Comment:
What uid is this?

Reply:
Jumping on it!

Reply:
Sorry for my total confusion. I see what you're talking about now. I have no idea now where I got that as it's the docker id and it's not what's referenced in the docker container setup guide: their default is 1000 for both.
I'll set it to 1000 now. If you think 0 is better, feel free to change it.

Reply:
I changed it to "0" to match the Transmission defaults to keep the two aligned

Reply:
The UID you specify needs to match one on the host system, so what they say/recommend is totally irellevant. 0 is good though 👍